ascend-sdk/src/main/java/ai/evolv/ExecutionQueue.java

```diff
@@ -11,7 +11,7 @@
 
 class ExecutionQueue {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(ExecutionQueue.class);
+    private static Logger LOGGER = LoggerFactory.getLogger(ExecutionQueue.class);
```

Comment:
Just curious, why did you have to remove `final`?

Reply:
No reason, it was accidentally copied over. Ill revert it.